front_end/entrypoints/rn_inspector/rn_inspector.ts

```diff
@@ -10,10 +10,21 @@ import '../inspector_main/inspector_main-meta.js';
 import '../../panels/issues/issues-meta.js';
 import '../../panels/mobile_throttling/mobile_throttling-meta.js';
 import '../../panels/timeline/timeline-meta.js';
+import '../../panels/rn_welcome/rn_welcome-meta.js';
 
 import * as Root from '../../core/root/root.js';
 import * as Main from '../main/main.js';
 
+Root.Runtime.experiments.register(
+  Root.Runtime.ExperimentName.REACT_NATIVE_SPECIFIC_UI,
+  'Show React Native-specific UI',
+  false,
```

Comment:
I've already forgotten what this boolean argument means, so let's add a comment.

```suggestion
  /* unstable */ false,
```